### PR TITLE
docs: add import subtree branch documentation

### PR DESCRIPTION
**Pull request by cbourgois**
### Description of the Change

21596fe03d — docs: add import subtree branch documentation

Preview 👀 : https://github.com/ovh-ux/manager/blob/docs/import-branch-subtree/DEVELOP.md#import-branch-from-legacy-repository

### CODE REVIEW

File: DEVELOP.md [hunk 1]
```diff
@@ -30,3 +30,63 @@ $ yarn run split --help
...
+git subtree pull --prefix=packages/manager/apps/telecom git@github.com:ovh-ux/ovh-manager-telecom.git feat/my-awesome-feature --squash
+```
+
+#### Resolve conflicts if present
+
+> Packages are renamed to `@ovh-ux/**` and are `private` now.
+>
+> Some dependencies have been updated (`ovh-angular-**` replaced by `@ovh-ux/ng-ovh-**`, `lodash` bumped to version 4), and all `eslint` errors & warnings are fixed.
+
+
```
Comment (FredericEspiau) [inline DEVELOP.md]: extra white line

File: DEVELOP.md [hunk 2]
```diff
@@ -30,3 +30,63 @@ $ yarn run split --help
...
+```bash
+git merge --continue
+```
+
+It's done ! :)
+
+
```
Comment (FredericEspiau) [inline DEVELOP.md]: extra white line

File: DEVELOP.md [hunk 3]
```diff
@@ -30,3 +30,63 @@ $ yarn run split --help
...
+| ------------- | ------------- | ------------- |
+| Manager Web  | `packages/manager/apps/web` | `git@github.com:ovh-ux/ovh-manager-web.git` |
+| Manager Dedicated  | `packages/manager/apps/dedicated` | `git@github.com:ovh-ux/ovh-manager-dedicated.git` |
+| Manager Cloud  | `packages/manager/apps/cloud` | `git@github.com:ovh-ux/ovh-manager-cloud.git` |
+| Manager Telecom  | `packages/manager/apps/telecom` | `git@github.com:ovh-ux/ovh-manager-telecom.git` |
+| Module Exchange  | `packages/manager/modules/exchange` | `git@github.com:ovh-ux/ovh-module-exchange.git` |
+| Module Sharepoint  | `packages/manager/modules/sharepoint` | `git@github.com:ovh-ux/ovh-module-sharepoint.git` |
+| Module Office  | `packages/manager/modules/office` | `git@github.com:ovh-ux/ovh-module-office.git` |
+| Module EmailPro  | `packages/manager/modules/emailpro` | `git@github.com:ovh-ux/ovh-module-emailpro.git` |
+
```
Comment (cbourgois) [inline DEVELOP.md]: Extra empty line

File: DEVELOP.md [hunk 4]
```diff
@@ -30,3 +30,59 @@ $ yarn run split --help
...
+### Create a branch from develop
+
+```bash
+git checkout -b feat/my-awesome-telecom-feature
+```
+
+### Import specific branch from legacy repo
+
+See [List of git subtree prefixes by repo](#list-of-git-subtree-prefixes-by-repo) for more details.
```
Comment (antleblanc) [inline DEVELOP.md]:
```suggestion
See [List of git subtree prefixes by repository](#list-of-git-subtree-prefixes-by-repository) for more details.
```

File: DEVELOP.md [hunk 5]
```diff
@@ -30,3 +30,59 @@ $ yarn run split --help
...
+
+```bash
+git subtree pull --prefix=packages/manager/apps/telecom git@github.com:ovh-ux/ovh-manager-telecom.git feat/my-awesome-feature --squash
+```
+
+### Resolve conflicts if present
+
+> Packages are renamed to `@ovh-ux/**` and are `private` now.
+>
+> Some dependencies have been updated (`ovh-angular-**` replaced by `@ovh-ux/ng-ovh-**`, `lodash` bumped to version 4), and all `eslint` errors & warnings are fixed.
```
Comment (antleblanc) [inline DEVELOP.md]:
```suggestion
> Some dependencies have been updated (`ovh-angular-**` replaced by `@ovh-ux/ng-ovh-**`, `lodash` bumped to version 4), and all ESLint errors & warnings are fixed.
```

File: DEVELOP.md [hunk 6]
```diff
@@ -30,3 +30,59 @@ $ yarn run split --help
...
+
+```bash
+git merge --continue
+```
+
+It's done ! :)
+
+### List of git subtree prefixes by repo
```
Comment (antleblanc) [inline DEVELOP.md]:
```suggestion
### List of git subtree prefixes by repository
```

File: DEVELOP.md [hunk 7]
```diff
@@ -30,3 +30,59 @@ $ yarn run split --help
...
+
+Use `PREFIX` and `REPO_URL` according the following table. `BRANCH_NAME` is the name of the legacy repository branch you want to import.
+
+Add `--squash` option if you want to squash commits.
+
+```bash
+git subtree pull --prefix=[PREFIX] [REPO_URL] [BRANCH_NAME] --squash
+```
+
+| Repo  | PREFIX  | REPO_URL |
```
Comment (antleblanc) [inline DEVELOP.md]:
```suggestion
| Repo  | PREFIX  | REPOSITORY_URL |
```

**Pull request merged**